packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -237,6 +290,7 @@ export const useNotifiSubscribe: ({
       // Mockup info for demo preview card
       setEmail(defaultDemoConfigV1.name);
       setPhoneNumber('+101234567890');
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       setTelegramId(defaultDemoConfigV1.id!);
```

Comment:
Adding the dummy discord for preview card might be a good idea: `setDiscrodId(defaultDemoConfigV1.id!)`

Reply:
yeah, i added a dummy code for the telegram id so avoid the nonnull assertion as well